src/ui/public/styles/spinner.less

```diff
@@ -1,39 +1,6 @@
-@import "~ui/styles/variables";
-
-@size: 400px;
-@color1: @kibanaPink1;
-@color2: @kibanaPink2;
-
 .spinner.ng-hide {
```

Comment:
is class='spinner' used anywhere anymore?  I didn't see it in the kibana repo but maybe it's in the kibana-framework repo.